tests/opentracer/test_tracer.py

```diff
@@ -71,6 +71,25 @@ def test_invalid_config_key(self):
             assert ["enabeld", "setttings"] in str(ce_info)
             assert tracer is not None
 
+    def test_global_tags(self):
+        """Global tags should be passed from the opentracer to the tracer."""
+        config = {
```

Comment:
`s/"/'/g`

Reply:
Yeah opted for consistency within the file here, will I run into `flake` issues?

Reply:
shouldn't, `flake8` doesn't check quotes.